packages/server/src/Socket.ts

```diff
@@ -36,5 +36,9 @@ export const createSocket = (server: HTTPServer) => {
     socket.on(GameEvent.UpdateState, (roomName: string, gameState: GameState) => {
       updateGameState(io, store, roomName, gameState);
     });
+
+    socket.on(GameEvent.PlayerAction, (roomName: string, playerAction: PlayerAction) => {
+      updateGameStateNew(io, store, roomName, playerAction);
```

Comment:
I guess this a provisory name right? :-p

Reply:
Yeah, I also said in the PR, this will substitute `updateGameState` but I wanted to do it step by step.